oluwajuwon/Fast-Food-Fast
# Create loading effect for frontend

### What does this pull request do?
This pull request adds loading effects to the platform for better user experience and also refactors part of the codebase

### Description of the Task to be completed?
 Users should know when a request has been made by seeing a loading spinner

### How should this be manually tested?
Users can test it by:
 - Install git
 - Open up the command line
 - Run `git clone https://github.com/oluwajuwon/Fast-Food-Fast.git` to clone the repository
 - Run `npm install` to install all package dependencies
 - Run `npm run start` to start the app
 - Visit http://localhost:3000/login.html to login as admin with the admin credentials
 - It should show a loader to let the user know that  request has been made

### What are the relevant pivotal tracker stories?
The relevant pivotal story is:
Admin should be able to decline an order - https://www.pivotaltracker.com/story/show/161705908
